### ISSUE TITLE
Color-contrast ratio is below 4.5:1 for DetailsRow checkbox in DetailsList 

### ISSUE THREAD

**Issue by shyama1208** [OfficeDev/office-ui-fabric-react#5076]
Color-contrast ratio is below 4.5:1 for checkbox for the row.
This causing Color test failure in Keros for Chrome.

![image](https://user-images.githubusercontent.com/26054709/40915311-325e93ca-6819-11e8-9e67-b0d22aa89e41.png)

**Comment by betrue-final-final** [comment]
For scan #1, that's hover and there should be a circle of #666666. 

The second scan is incorrect  and a limitation in automation. The white outline it's registering as noncompliant is for the visibility against the blue fill, but Keros is comparing it against the grey. One must always use human judgment when using automation tools.

**Comment by michaelangotti** [comment]
@Jahnp Can you assign this to one of your designers?

**Comment by Jahnp** [comment]
@michaelangotti @betrue-final-final sorry, what's the action needed here?

**Comment by betrue-final-final** [comment]
We just need to update the check circle to be neutralSecondary (#666666) instead of neutralTertiaryAlt (#c8c8c8).

The other issue is an incorrect assessment by Keros due to how the check is drawn.
